cmd/eksctl-anywhere/cmd/applypackages.go

```diff
@@ -42,7 +51,12 @@ var applyPackagesCommand = &cobra.Command{
 }
 
 func applyPackages(ctx context.Context) error {
-	kubeConfig := kubeconfig.FromEnvironment()
+	kubeConfig := lpo.kubeConfig
```

Comment:
apo?

Reply:
Yep, I managed to push the wrong version. My bad. Let me push the right one.